src/models/train_model.py

```diff
@@ -4,86 +4,41 @@
 import glob
 import json
 import os
+from pathlib import Path
 
 import numpy as np
 import setGPU  # noqa: F401
 import torch
 import torch.nn as nn
 import torch.optim as optim
 import tqdm
+import yaml
 
 from src.data.h5data import H5Data
 from src.models.models import GraphNet
 
 os.environ["HDF5_USE_FILE_LOCKING"] = "FALSE"
-test_path = "dataset/test/"
-train_path = "dataset/train/"
-NBINS = 40  # number of bins for loss function
-MMAX = 200.0  # max value
-MMIN = 40.0  # min value
-
-N = 60  # number of charged particles
-N_sv = 5  # number of SVs
-n_targets = 2  # number of classes
-device = "cpu"
```

Comment:
This device variable is not used. Moreover, model.py used .cuda() many times, so we need to add a cpu option if time permits